src/Kodify/BlogBundle/Resources/config/routing.yml

```diff
@@ -1,5 +1,8 @@
 home:
     path:     /
+    defaults: { _controller: KodifyBlogBundle:Posts:index, sort: 1 }
+sorted_home:
+    path:     /sort/{sort}
```

Comment:
Hey Miquel, did you notice that accessing to "/sort/1" would be the same as "/"?

Could you comment here which would be the way to avoid that duplicity?

Cheers! :)